doc/source/workflows/advanced.rst

```diff
@@ -1,58 +1,11 @@
 Advanced Topics
 ===============
 
-Inplace Execution
------------------
-
-When executing a workflow task inside another workflow task, it is usually executed in another Ray worker process. This is good for resource and performance isolation, but at the cost of lower efficiency due to non-locality, scheduling and data transfer.
-
-For example, this recursive workflow calculates the exponent. We write it with workflow so that we can recover from any task. However, it is really inefficient to scheduling each task in a different worker.
-
-.. code-block:: python
-    :caption: Workflow without inplace execution:
-
-    import ray
-    from ray import workflow
-
-    @ray.remote
-    def exp_remote(k, n):
-        if n == 0:
-            return k
-        return workflow.continuation(exp_remote.bind(2 * k, n - 1))
-
-We could optimize it with inplace option:
-
-.. code-block:: python
-    :caption: Workflow with inplace execution:
-
-    import ray
-    from ray import workflow
-
-    @ray.remote
-    def exp_inplace(k, n):
-        if n == 0:
-            return k
-        return workflow.continuation(
-            exp_inplace.options(**workflow.options(allow_inplace=True)).bind(2 * k, n - 1))
-
-    assert workflow.create(exp_inplace.bind(3, 7)).run() == 3 * 2 ** 7
-
-
-With ``allow_inplace=True``, the task that called ``.bind()`` executes in the function. Ray options are ignored because they are used for remote execution. Also, you cannot retrieve the output of an inplace task using ``workflow.get_output()`` before it finishes execution.
-
-Inplace is also useful when you need to pass something that is only valid in the current process/physical machine to another task. For example:
-
-.. code-block:: python
-
-    @ray.remote
-    def foo():
-        x = "<something that is only valid in the current process>"
-        return workflow.continuation(bar.options(**workflow.options(allow_inplace=True)).bind(x))
-
-
 Wait for Partial Results
 ------------------------
 
+**NOTE: workflow.wait() is temporarily removed due to workflow refactoring. We will add it back later.**
```

Comment:
Maybe just delete it temporarily and add it back once we make it work?

Reply:
ok